image_provisioner/playbooks/delete_oldest.yaml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Delete the oldest file in the directory
+  hosts: localhost
+  roles:
```

Comment:
Instead of a separate playbook, can you call this role from existing playbooks?

Reply:
Also can you make this role optional?

Reply:
@ravi yes we can do that. I will add it to the provision_gold_images.yaml playbook.